package com.goodforgoodbusiness.endpoint.plugin.internal;

public class InternalReasonerException extends Exception {
	public InternalReasonerException(String message, Throwable cause) {
		super(message, cause);
	}
}
